python/servo/post_build_commands.py

```diff
@@ -75,7 +75,11 @@ def run(self, params, release=False, debug=False, debugger=None):
         else:
             args = args + params
 
-        subprocess.check_call(args, env=env)
+        try:
+            subprocess.check_call(args, env=env)
+        except OSError:
```

Comment:
Does an `OSError` include when the program was terminated, or had a nonzero return?

I'd much rather prefer checking for the existence of the binary, and then running `check_call`